dashboard/app/views/sections/show.html.haml

```diff
@@ -98,10 +104,11 @@
         %h4.instructions= t('signinsection.words')
         = text_field_tag :secret_words, nil, autocomplete: 'off'
         .clear
-    #pairing_checkbox{style: 'display: none'}
-      = check_box_tag :show_pairing_dialog
-      = label_tag :show_pairing_dialog, t('signinsection.pair_programming')
-    = button_tag t('signinsection.login'), id: 'login_button', style: 'display:none', class: 'btn btn-primary'
+
+        = button_tag t('signinsection.login'), id: 'login_button', style: 'display:none', class: 'btn btn-primary'
```

Comment:
Are you putting a sign in button both above and below the pictures?

Reply:
Talked it over with Ryan and he decided to have it above the secret pictures only.  I'm happy to do it either way though!